cmd/manager/main.go

```diff
@@ -128,8 +129,10 @@ func main() {
 
 	// Create a new Cmd to provide shared dependencies and start components
 	mgr, err := manager.New(cfg, manager.Options{
-		Namespace:          namespace,
-		MapperProvider:     restmapper.NewDynamicRESTMapper,
+		Namespace: namespace,
+		MapperProvider: func(c *rest.Config) (meta.RESTMapper, error) {
+			return apiutil.NewDynamicRESTMapper(c)
+		},
```

Comment:
One question I forgot to ask last time. Does this work?
```suggestion
		MapperProvider: apiutil.NewDynamicRESTMapper,
```

Reply:
I would think so, but:

`cmd/manager/main.go:133:3: cannot use apiutil.NewDynamicRESTMapper (type func(*rest.Config, ...apiutil.DynamicRESTMapperOption) (meta.RESTMapper, error)) as type func(*rest.Config) (meta.RESTMapper, error) in field value`

Reply:
Ah makes sense. Thanks